echo-station/echo-station
# Basic Wheelchair Functionality

### Summary
Added the 'Low Mobility' trait, which reduces move speed but comes with a 'Wheelchair' which can be buckled into.

### How To Test:
1. Add the 'Low Mobility' trait to a character.
2. Spawn as that character.
3. Attempt to move.
4. Observe movement is achieved at a reduced speed.
5. Observe a wheelchair is present in the character's inventory.
6. Place the wheelchair on the ground by pressing 'Q'.
7. Right click on the wheelchair and buckle into it.
8. Attempt to move.
9. Observe wheelchair movement is achieved at a speed faster than player speed.

### Quirks
* The functionality added by this PR are the wheelchair-related stories with the highest perceived value, but additional features are planned (sounds, upgrades, storage, medical fabricator recipe, etc.).
* Left clicking on a wheelchair without being strapped into it picks up the wheelchair. This behavior is planned to be changed in a future PR.
* Low Mobility & Wheelchair speeds are subject to change.
* Currently, no entities are blacklisted from wheelchairs.

## Changelog
:cl: Over
add: Added "Wheelchair", which can be piloted by buckled entities.
add: Added "Low Mobility" trait, which reduces base move speed and comes with a wheelchair.
